montepy/mcnp_object.py

```diff
@@ -100,22 +101,34 @@ class MCNP_Object(ABC, metaclass=_ExceptionContextAdder):
         For init removed ``comments``, and added ``parser`` as arguments.
 
     :param input: The Input syntax object this will wrap and parse.
-    :type input: Input
+    :type input: Union[Input, str]
     :param parser: The parser object to parse the input with.
-    :type parser: MCNP_Lexer
+    :type parser: MCNP_Parser
+    """
+
+    """
+    The block type this input comes from.
     """
```

Comment:
Is this separate from the docstring intentionally?

Reply:
Oh I need to clean that up. It was a doc string for a class variable I had to move.